### PR TITLE
added new scope

### PR DESCRIPTION
**Pull request by japoorv**
Added new `SENSITIVE_SPAN` scope for sensitive data attributes.

### CODE REVIEW

**Review by avinashkolluru — APPROVED** [review]

File: attribute-service-api/src/main/proto/org/hypertrace/core/attribute/service/v1/attribute_metadata.proto [hunk 1]
```diff
@@ -28,6 +28,7 @@ enum AttributeScope {
   CLUSTERS_SNAPSHOT = 17;
   DOMAIN = 18;
   ACTOR = 19;
+  SPAN_SENSITIVE_DATA = 20;
```
Comment (skjindal93) [inline attribute-service-api/src/main/proto/org/hypertrace/core/attribute/service/v1/attribute_metadata.proto:31]: Can we rename it to `SENSITIVE_SPAN` or `SENSITIVE_EVENT`? We interchangeably use `SPAN` and `EVENT` in attribute scopes

Comment (avinashkolluru) [reply attribute-service-api/src/main/proto/org/hypertrace/core/attribute/service/v1/attribute_metadata.proto:31]: No EVENT please. EVENT was wrongly named. It should have been SPAN. So lets not continue to name new enums that way.

**Review by skjindal93 — APPROVED** [review]

File: .snyk [hunk 2]
```diff
@@ -5,5 +5,5 @@ ignore:
   SNYK-JAVA-IONETTY-1042268:
     - '*':
         reason: No replacement available
-        expires: 2022-04-30T00:00:00.000Z
+        expires: 2022-05-30T00:00:00.000Z
```
Comment (avinashkolluru) [inline .snyk:8]: Isn't this vulnerability fixed with the upgrade?

Comment (japoorv) [reply .snyk:8]: The initial issue was the expiration one, the other one was due to netty issue.
Upgraded the version, everything fine now

Comment (japoorv) [reply .snyk:8]: Removing this now

**Review by avinashkolluru — APPROVED** [review]

**Pull request merged**